ruby/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] Please make sure to add the rentals to the associations in the `Book` class as well